README.md

```diff
@@ -226,41 +313,81 @@ But there are three important differences between a proxy reducer and an ordinar
-
-| `yarn start:grid`                             | `yarn start:todo`                              | `yarn start:chat`                             |
-| --------------------------------------------- | ---------------------------------------------- | --------------------------------------------- |
-| A simple table editor                         | An implementation of TodoMVC                   | A chat channel                                |
-| <img src='images/screen.grid.png' width=600/> | <img src='images/screen.todo.png'  width=600/> | <img src='images/screen.grid.png' width=600/> |
-
-If you are working on the Cevitxe code and/or the code for one of the demo apps, you can start in
-watch mode by running `yarn dev` instead:
-
-- `yarn dev:grid`
-- `yarn dev:todo`
-- `yarn dev:chat`
-
-In each case the app will run on `localhost:3000`. (You can only run one example app at a time).
-
-To test the peer-to-peer functionality on a single computer, visit the demo in two different
-browsers (e.g. Chrome and Firefox), or with a normal window and an incognito window of the same
-browser.
-
-(You can of course run in two normal tabs of the same browser, but they'll share local
-state so you're not really testing their ability to communicate.)
-
-Copy the ID from one browser window and paste it into the other, then press **Join**.
-After a brief delay, you should have the same state visible in both instances, and changes made in
-one should be replicated to the other.
-
-![](images/demo.gif)
-
-## Known limitations
-
-- Performance starts to degrade with datasets over 100K rows (TODO: narrow this down)
+## Limitations
+
+Cevitxe requires that the entire repository be present on each peer's machine. That means that it is
+limited to datasets that can fit comfortably within the disk space on a single computer. In 2019,
+that means something on the order of 1-10 GB.
+
+## Work in progress
+
+- [ ] **Run in worker process**  
+       Enable running Cevitxe's `StoreManager` entirely in a web worker process. All the Automerge,
+      local storage and sync components would live in the worker, thus freeing the UI context of all the
+      expensive overhead of the system. We'll need a coarse-grained query API to query and retrieve
+      a view of a set of rows, aggregate information, etc. #45
+  - [ ] order + range / count
+  - [ ] filter
+  - [ ] aggregation, projection, etc. (might be entirely out of scope)
+
+* [ ] **Deployable always-on superpeer**  
+       In a peer-to-peer network where clients are only online intermittently and updates can only
+      happen when two peers happen to be online at the same time, it can take a while for updates to
+      propagate. One solution would be to leave a client running on a computer that's never turned
+      off. We'd like to provide a "superpeer" (don't call it a server!!) that can be inexpensively
+      deployed using the command line to cloud hosting providers. #41
```

Comment:
We could borrow P2P sharing terminology and call it a "seeding peer" or something like that (IDC)